examples/stable-diffusion-jax/example.py

```diff
@@ -0,0 +1,89 @@
+import jax
```

Comment:
What's the difference among `alpa_run.py`, `example.py`, `nopmap.py` and `run.py`? If some are not related to alpa, could you please clean them up?